docs/apis-tools/tasklist-api-rest/schemas/requests/task-search-request.mdx

```diff
@@ -156,6 +164,10 @@ TaskSearchRequest - query object to search tasks by provided params.
 
 > Specifying a range of due dates for the tasks to search for.
 
+#### [<code style={{ fontWeight: 'normal' }}>TaskSearchRequest.<b>taskVariables</b></code>](#)<Bullet />[`TaskVariables`](../models/task-variables.mdx) <Badge class="secondary" text="list"/>
+
+> An array of filter clauses specifying the variables to filter for.  If defined, the query returns only tasks to which all clauses apply.. However, it's important to note that this filtering mechanism is designed to work exclusively with truncated variables. This means that variables of a larger size are not compatible with this filter, and attempts to use them may result in inaccurate or incomplete query results.
```

Comment:
Typo (two ".." after apply), otherwise good!

Reply:
Just fixed :) Can you reapprove ? Thanks